src/gui/accountmanager.cpp

```diff
@@ -254,8 +254,11 @@ void AccountManager::saveAccountHelper(Account *acc, QSettings &settings, bool s
             // re-persisting them)
             acc->_credentials->persist();
         }
-        Q_FOREACH (QString key, acc->_settingsMap.keys()) {
-            settings.setValue(key, acc->_settingsMap.value(key));
+
+        auto i = acc->_settingsMap.constBegin();
+        while (i != acc->_settingsMap.constEnd()) {
```

Comment:
I'd prefer `for(auto it = acc->_settingsMap.constBegin(); it != acc->_settingsMap.constEnd(); ++it)`

Reply:
Done! :)